doc/en/assert.rst

```diff
@@ -85,6 +85,13 @@ and if you need to have access to the actual exception info you may use::
 the actual exception raised.  The main attributes of interest are
 ``.type``, ``.value`` and ``.traceback``.
 
+In the context manager form you may use the keyword argument
```

Comment:
> Also wouldn't a versionchanged be more appropriate as the function was changed?

Good point. I was thinking along the lines of *adding a new parameter*, but I think your suggestion makes more sense.